database/create-database.sql

```diff
@@ -35,6 +35,7 @@ CREATE TABLE reply (
 	reply VARCHAR (200) NOT NULL,
     id_topic INT,
     creation_date_reply TIMESTAMP NOT NULL,
+    deleted_reply TINYINT NOT NULL,
```

Comment:
Shouldn't this be a boolean?...

Reply:
I used TINYINT, because has more compatibility with other systems, some database management systems dont support a BOOLEAN data type.